<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDataDlg</class>
 <widget class="QDialog" name="UpdateDataDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoapTyping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../hla.qrc">
    <normaloff>:/png/images/about.png</normaloff>:/png/images/about.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>598</width>
     <height>348</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Selective Update</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Gene File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Allele File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Gssp File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Lab Align File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Common File:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gene">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Allele">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gssp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_LabAlign">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Common">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGene">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAllele">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGssp">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLabAlign">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCommon">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnApply">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Insert GSSP</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GSSP Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Gene Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Exon Index[1,8]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gssp_Name">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gene_Name">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>40</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Exon">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>40</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FR">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>110</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Base">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>110</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>F or R(F/R):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GSSP Bases:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Pos">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>110</y>
       <width>185</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GSSP Position:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsert">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../hla.qrc"/>
 </resources>
 <connections/>
</ui>
